### PR TITLE
Simplify template

### PR DESCRIPTION
**Pull request by adamwight**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -122,30 +122,33 @@ include  Makefile.manual
 datasets/arwiki.sampled_revisions.20k_2016.json:
 	wget -qO- http://quarry.wmflabs.org/run/65713/output/0/json-lines?download=true > $@
 
-datasets/arwiki.autolabeled_revisions.20k_2016.json: \
+datasets/arwiki.autolabeled_revisions.20k_2016.review.json \
+datasets/arwiki.autolabeled_revisions.20k_2016.no_review.json: \
```
Comment (halfak) [inline Makefile]: This looks wrong.

File: editquality/utilities/autolabel.py [hunk 2]
```diff
@@ -39,7 +40,10 @@
                              processing edits. [default: <cpu_count>]
     --input=<path>           The path to a file containing rev_ids to process
                              [default: <stdin>]
-    --output=<path>          The path to write labels to [default: <stdout>]
+    --output-needs-review=<path> The path to write observations in need of review.
+                                 [default: <stdout>]
+    --output-no-review=<path>    The path to write assumed good observations.
+                                 [default: <stdout>]
```
Comment (halfak) [inline editquality/utilities/autolabel.py]: This seems overly complicated.  I don't see why filtering after the fact is any more difficult and it's certainly less messy to allow the filter logic to be modular.

**Comment by adamwight** [comment]
I'm comparing model performance before and after this PR:
```
revscoring model_info models/cswiki.damaging.gradient_boosting.model > master_cswiki_info
rm datasets/cswiki.human_labeled_revisions.5k_2016.* datasets/cswiki.labeled_revisions.*
git checkout -b simplify_template origin/simplify_template
rm models/cswiki.damaging.gradient_boosting.model
make models/cswiki.damaging.gradient_boosting.model
```

**Comment by adamwight** [comment]
Dramatically worse.  I think the main issue is that we no longer have "true" predictions:
```
-       counts (n=17914):
+       counts (n=15468):
                label        n         ~True    ~False
                -------  -----  ---  -------  --------
-               True       451  -->      221       230
-               False    17463  -->      395     17068
+               True        50  -->        0        50
+               False    15418  -->        0     15418
```

**Comment by adamwight** [comment]
Model statistics look similar.  This patch fixes some duplicated observations, which affects the stats as well.

Here are the diffs in model_info:
https://phabricator.wikimedia.org/P6878 (goodfaith)
https://phabricator.wikimedia.org/P6879 (damaging)

**Pull request merged**